plane/source/docs/guide-tilt-rotor.rst

```diff
@@ -221,6 +221,26 @@ degrees that the tilt mechanism can go.
 You also need to setup your two tilt servos with SERVOn_FUNCTION=75
 for left tilt and SERVOn_FUNCTION=76 for right tilt.
 
+Non-Vectored Yaw
+================
+
+Non-Vectored yaw aircraft (Q_TILT_TYPE 0 or 1) needs a tilt servo for the rear
+motor for yaw control.
+
+You need to setup your two front tilt servos both with SERVOn_FUNCTION=41
+and also your rear servo for yaw control with SERVOn_FUNCTION=39. 
+FUNCTION=39 is normally the servofunction for motor 7. Ony in non-vectored
+tri-copter the yaw servo is controlled via SERVOn_FUNCTION 39.
+
+Note that if you want to use BLHELI ESCs with telemetry in non-vectored tri-copter,
+you must not set BLHELI to 1. Set instead SERVO_BLH_MASK to the output-bitmask
+of the servo-channels actually connected BLHELI-ESCs.
+
+For example if your motors are conntected to servo 9,10,11 (the first three aux-outputs of a pixhawk1), 
+set SERVO_BLH_MASK to 1792. 
+
+You also need to setup Q_M_PWM_TYPE to 4 (DShot 150).
```

Comment:
is DShot 150 required? is it needed for esc telemetry? 
I believe that you can use BLHELI escs without even having the SERVO_BLH_MASK set or using SERVO_BLH_AUTO..only if you wish to use passthrough programming on BLH32 escs do you need to set these params...and I believe telemetry works even if using other motor control protocols. like PWM or Oneshot, but I could be mistaken..

Reply:
So far I thought that the DSHot is necessary for telemetry. Since you're asking, I'm not sure.
Of course you can use BLHEILI32, BLHEILI32 with DSHOT in non-vectored Tris. 
The problem only occurs when SERVO_BLH_AUTO is 1.

Reply:
I did some research....for telem to work, you must use one of the DShot protocols, since we recommend 150, the statement is correct....however, I suggest this:
"If you wish to also setup BLEHeli esc telemetry, you need to set Q_M_PWM_TYPE to 4 (DShot 150)"